lib/routes/authorization.js

```diff
@@ -142,6 +146,9 @@ module.exports = {
         .valid(ACCESS_TYPE_OFFLINE, ACCESS_TYPE_ONLINE)
         .default(ACCESS_TYPE_ONLINE)
         .optional(),
+      // PKCE support
+      code_challenge_method: Joi.string().optional(), // Force 'S256'
```

Comment:
@rfk thoughts on only supporting the `S256` and not `plain` - code challenge method?